src/components/Hamburger/Hamburger.js

```diff
@@ -0,0 +1,145 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled, { css } from 'react-emotion';
+import { hideVisually } from 'polished';
+
+const HAMBURGER_WIDTH = '16px';
+const LINE_HEIGHT = '2px';
+
+const buttonBaseStyles = ({ theme }) => css`
+  label: hamburger-button;
+  box-sizing: content-box;
+  cursor: pointer;
+  display: inline-block;
+  padding: ${theme.spacings.mega} ${theme.spacings.byte};
+  width: ${HAMBURGER_WIDTH};
+  background: none;
+  border: 0;
+  position: relative;
+`;
+
+const layersBaseStyles = ({ theme }) => css`
+  label: hamburger-layers;
+  margin-top: calc(${LINE_HEIGHT} / -2);
+  top: 50%;
+  width: calc(${HAMBURGER_WIDTH} * 0.77);
+  transition: 0.3s cubic-bezier(0.55, 0.055, 0.675, 0.19);
+
+  &,
+  &::after,
+  &::before {
+    background-color: ${theme.colors.n900};
+    border-radius: 1px;
+    display: block;
+    height: ${LINE_HEIGHT};
+    position: absolute;
+    transition: width 0.2s ease-out 0.14s, top 0.1s 0.34s ease-in,
```

Comment:
If you use only `opacity` and `transform` inside `transition` you can archieve better and smooth animations.
Because only these two properties run on Composite step in the browser render.

I was trying to find an article to explain this and maybe this is good one :)
https://medium.com/outsystems-experts/how-to-achieve-60-fps-animations-with-css3-db7b98610108

Reply:
I ported this component from the website — it's very old and probably predates CSS3 😉 
Thanks for pointing out this opportunity for improvement!